src/sorting/Sort.ts

```diff
@@ -61,8 +61,21 @@ export const selectionSort = <T>(items: T[]): T[] => {
 }
 
 export const insertionSort = <T>(items: T[]): T[] => {
-  throw new Error('Not Implemented!')
-  return []  
+  let output = clone(items)
+  
+  for (let i=1; i < output.length; i++){
+     let currVal = output[i], j = i - 1;
```

Comment:
Don't initialize multiple variables on a single line. We're going to configure linting and formatting tools soon that will disallow and or fix that. But just fyi - that is not good practice in JS/TS.